### ISSUE TITLE
Upgrading multireducer to v3.0.3

### ISSUE THREAD

**Issue by Nishant-Pathak** [erikras/multireducer#121]
Hi @erikras 
I have used react redux universal template for my few project and found it very useful. I would like to update the dependencies. Can you guide me for updating multireducer. Lets take [/src/components/CounterButton/CounterButton.js](https://github.com/erikras/react-redux-universal-hot-example/blob/master/src/components/CounterButton/CounterButton.js) class.

**Comment by yesmeck** [comment]
@Nishant-Pathak You can find migration guide here https://github.com/erikras/multireducer/releases/tag/v3.0.0-beta2

**Comment by Nishant-Pathak** [comment]
Yeah I have seen that, I was not able to understand much from it (annotation processing etc).

**Comment by yesmeck** [comment]
Hello @Nishant-Pathak , I've send a PR to react-redux-universal-hot-example https://github.com/erikras/react-redux-universal-hot-example/pull/1343.

**Comment by Nishant-Pathak** [comment]
Nice, @yesmeck lets get it merge :+1:
